tone-analyzer/v3-generated.ts

```diff
@@ -115,8 +114,9 @@ class ToneAnalyzerV3 extends BaseService {
    *
    * @param {Object} params - The parameters to send to the service.
    * @param {Utterance[]} params.utterances - An array of `Utterance` objects that provides the input content that the service is to analyze.
-   * @param {string} [params.content_language] - The language of the input text for the request: English or French. Regional variants are treated as their parent language; for example, `en-US` is interpreted as `en`. The input content must match the specified language. Do not submit content that contains both languages. You can specify any combination of languages for `Content-Language` and `Accept-Language`. * **`2017-09-21`:** Accepts `en` or `fr`. * **`2016-05-19`:** Accepts only `en`.
-   * @param {string} [params.accept_language] - The desired language of the response. For two-character arguments, regional variants are treated as their parent language; for example, `en-US` is interpreted as `en`.
+   * @param {string} [params.content_language] - The language of the input text for the request: English or French. Regional variants are treated as their parent language; for example, `en-US` is interpreted as `en`. The input content must match the specified language. Do not submit content that contains both languages. You can use different languages for **Content-Language** and **Accept-Language**. * **`2017-09-21`:** Accepts `en` or `fr`. * **`2016-05-19`:** Accepts only `en`.
+   * @param {string} [params.accept_language] - The desired language of the response. For two-character arguments, regional variants are treated as their parent language; for example, `en-US` is interpreted as `en`. You can use different languages for **Content-Language** and **Accept-Language**.
+   * @param {Object} [params.headers] - Custom request headers
```

Comment:
When describing the GDPR support please make sure you communicate that users may override headers that we use as parameters.

If they send {

```js
{ params: {
    accept_language: 'en'
    headers: {
      'Accept-Language': 'batman'
    }
  }
}
```
Batman will take presedence.

```js
      defaultOptions: extend(true, {}, this._options, {
        headers: {
        headers: extend(true, {
          'Accept': 'application/json',
          'Content-Type': 'multipart/form-data',
        }
      })
        }, _params.headers),
      }),
```

Reply:
Ok, I updated the README.